gridstatus/lmp_config.py

```diff
@@ -0,0 +1,154 @@
+import datetime
+import functools
+import inspect
+
+import pandas as pd
+
+from gridstatus.base import ISOBase, Markets, NotSupported
+
+
+class lmp_config:
```

Comment:
we can do it in a separate PR, but I assume this decorator could also support validating `locations` and `location_type` parameters. maybe we can file an issue or leave a todo in the code?

Reply:
#137